ra2ce/analysis/adaptation/adaptation_option.py

```diff
@@ -155,4 +157,9 @@ def calculate_impact(self, benefit_graph: GeoDataFrame) -> GeoDataFrame:
         _option_cols = benefit_graph.filter(regex=f"{self.id}_").columns
         benefit_graph[f"{self.id}_impact"] = benefit_graph[_option_cols].sum(axis=1)
 
+        # convert event impact into time-horizon impact
+        benefit_graph[f"{self.id}_impact"] = (
+            benefit_graph[f"{self.id}_impact"] * net_present_value_factor
+        )
```

Comment:
Not required, but I think you can do:
```suggestion
        benefit_graph[f"{self.id}_impact"] *= net_present_value_factor
        )
```